.gitignore

```diff
@@ -136,7 +136,6 @@ dist
 .DS_Store
 
 # lock files
-package-lock.json
```

Comment:
This line should be present.

Reply:
I took a look at different repos at fastify and pino. I realised we don't have lock file in most of the repos. Why is that we don't want to have that to track the exact version of packages?

Reply:
A package lock provides no benefit in a library project. It is ignored when the library is installed as a dependency.